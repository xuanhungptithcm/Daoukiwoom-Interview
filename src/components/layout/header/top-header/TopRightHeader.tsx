import { ReactElement } from "react";
import { ReactComponent as DownIconSVG } from "../../../../assets/icons/svg/down.svg";
import "./topHeader.scss";

const TopRightHeader = (): ReactElement => {
  return (
    <div className="container-top-header">
      <div className="top-menu">
        <ul className="menus">
          <li className="menu-item">
            <span>EN</span>
            <DownIconSVG />
          </li>
          <li className="menu-item">
            <span>Company</span>
            <DownIconSVG />
          </li>
          <li className="menu-item">
            <span>Partner Market Place</span>
            <DownIconSVG />
          </li>
        </ul>
      </div>
    </div>
  );
};

export default TopRightHeader;
