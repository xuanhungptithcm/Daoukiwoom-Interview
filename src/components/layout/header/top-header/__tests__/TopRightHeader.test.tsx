/**
 * @jest-environment jsdom
 */

import { shallow } from "enzyme";
import TopRightHeader from "../TopRightHeader";

describe("TopRightHeader", () => {
  test("should properly render", () => {
    expect(shallow(<TopRightHeader />)).toMatchSnapshot();
  });
});
