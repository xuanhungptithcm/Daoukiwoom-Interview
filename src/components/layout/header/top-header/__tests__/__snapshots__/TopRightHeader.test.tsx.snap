// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopRightHeader should properly render 1`] = `
<div
  className="container-top-header"
>
  <div
    className="top-menu"
  >
    <ul
      className="menus"
    >
      <li
        className="menu-item"
      >
        <span>
          EN
        </span>
        <ReactComponent />
      </li>
      <li
        className="menu-item"
      >
        <span>
          Company
        </span>
        <ReactComponent />
      </li>
      <li
        className="menu-item"
      >
        <span>
          Partner Market Place
        </span>
        <ReactComponent />
      </li>
    </ul>
  </div>
</div>
`;
